src/App.css

```diff
@@ -1,39 +1,56 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
-.App-header {
-  background-color: #282c34;
+body {
+  background-color: purple;
   min-height: 100vh;
   display: flex;
-  flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  font-family: 'Roboto', sans-serif;
 }
 
-.App-link {
-  color: #61dafb;
+.calculator {
+  background-color: #fff;
+  height: 33rem;
+  width: 23.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+button {
+  height: 5.5rem;
+  width: 5.875rem;
+  font-size: 1.5rem;
+  background-color: #e3e3e3;
+  cursor: pointer;
```

Comment:
- [ ] It would be better if our design matches up with the required design. The borders are visibly seen on our project, so I would suggest applying `border: none;` to the button and let see if we are closer to achieving the required design

### Our Design
![Screenshot from 2022-08-16 23-09-20](https://user-images.githubusercontent.com/46388113/185000984-4c82fdf3-8da7-4e9c-a3b4-3b68ea557fa4.png)


### Require Design

![Screenshot from 2022-08-16 23-09-52](https://user-images.githubusercontent.com/46388113/185000914-24198fe5-9fee-4214-9449-4c84f54a5fba.png)
 Let us apply some margin to see if we can